packages/calling-stateful-client/src/StreamUtils.ts

```diff
@@ -34,21 +35,27 @@ async function createViewRemoteVideo(
   } else {
     participantKey = toFlatCommunicationIdentifier(participantId);
   }
+  const streamLogInfo = `callId: ${callId}, participantKey: ${participantKey}, streamId: ${streamId}`;
+
+  callingStatefulLogger.info(`Start creating view for remote video! ${streamLogInfo}`);
```

Comment:
Extra question (related to my other question) is this enabled by default for all customers?